pkg/app/app.go

```diff
@@ -85,6 +86,34 @@ func initConfiguration(config configuration.Configuration, apiClient api.ApiClie
 		return orgId
 	})
 
+	config.AddDefaultValue(configuration.ORGANIZATION_SLUG, func(existingValue any) any {
+		client := networking.NewNetworkAccess(config).GetHttpClient()
+		url := config.GetString(configuration.API_URL)
+		apiClient.Init(url, client)
+		if existingValue != nil && len(existingValue.(string)) > 0 {
+			orgName := existingValue.(string)
+			_, err := uuid.Parse(orgName)
+			isUUID := err == nil
+			if isUUID {
+				orgName, err = apiClient.GetSlugFromOrgId(existingValue.(string))
+				if err != nil {
+					logger.Print("Failed to determine default value for \"ORGANIZATION_SLUG\":", err)
+				} else {
+					return orgName
+				}
+			} else {
+				return orgName
+			}
+		}
+
+		orgId, err := apiClient.GetDefaultOrgId()
```

Comment:
Issue: here the default org id is returned, which should actually be the default org slug name.